doc/COMPILE.md

```diff
@@ -324,7 +324,8 @@ The following base flags are available:
 
 The user flags are (the order matters!):
 
-- `p` use passwd/ldap to lookup home directory using `getpwuid_r`
+- `p` use passwd/ldap to lookup home directory using `getpwnam_r` with the username from the POSIX environment variable LOGNAME
+- `s` same as `p` but with the environment variable SUDO_USER, which enables access to the `user:/` namespace of the executing user when using `sudo`
```

Comment:
Please also support `DOAS_USER`. Please move this documentation to the resolver and link from here to the resolver.

Reply:
done.